src/app/pages/administration/search/landing-page.jsx

```diff
@@ -134,15 +137,15 @@ function ChplLandingPage() {
       <Box className={classes.landingPageBackground} pb={16} height="fit-content">
         <Box className={classes.landingPageImageryBackground}>
           <Container maxWidth="md">
-            <Box className={classes.searchContainer} mt={-8}>
+            <Box className={classes.searchContainer}>
               <ChplFilterSearchTerm />
-              <ChplFilterPanel />
-              <ChplFilterQuickFilters />
+              <Box display="flex" justifyContent="space-between">
+                <ChplFilterPanel />
+                <ChplFilterQuickFilters />
+              </Box>
             </Box>
-            <Box pt={8} pb={4}>
-              <Typography className={classes.subHeaders} align="left" variant="h2" gutterBottom>
-                Use our collections pages to help find a particular category of listings
-              </Typography>
+            <Box pb={4}>
+              <Typography className={classes.subHeaders} align="left" variant="h2" gutterBottom>Use our collections pages to help find a particular category of listings</Typography>
```

Comment:
```suggestion
              <Typography className={classes.subHeaders} align="left" variant="h2" gutterBottom>
                  Use our collections pages to help find a particular category of listings
              </Typography>
```

I'm pretty sure our standard on Typography blocks is to have the actual paragraph texts in a separate line from the defining blocks. That really helps distinguish between changes to the styles / element definitions and changes to the text contents